render/render.go

```diff
@@ -178,10 +178,12 @@ type Filter struct {
 // Render implements Renderer
 func (f Filter) Render(rpt report.Report) RenderableNodes {
 	output := RenderableNodes{}
+	inDegrees := map[string]int{}
 	for id, node := range f.Renderer.Render(rpt) {
 		if f.FilterFunc(node) {
 			output[id] = node
 		}
+		inDegrees[id] = 0
```

Comment:
Is this like the initial set? If so, it's not necessary: the zero value of an int is zero.